src/bin/schema.rs

```diff
@@ -0,0 +1,10 @@
+use cosmwasm_schema::write_api;
+use financial_snapshot_contract::msg::{ExecuteMsg, InstantiateMsg, QueryMsg};
```

Comment:
there's no ExecuteMsg define in your code

Reply:
Got it.

Reply:
Done.